frontend/src/metabase/visualizations/visualizations/PieChart.jsx

```diff
@@ -288,6 +288,7 @@ export default class PieChart extends Component {
     const formatPercent = percent =>
       formatValue(percent, {
         column: cols[metricIndex],
+        number_separators: settings.column(cols[metricIndex]).number_separators,
```

Comment:
```suggestion
        ...settings.column(cols[metricIndex]),
```

I think this change will work, but it's worth testing. `settings.column()`'s result should have `column` in it.

It's probably identical but might save us from a similar oversight in the future.

Reply:
Oh, I tried this explicitly, it removed the percentage signs in the percent, I tried for a few hours to get it to work with the other column stuff and it does seem that I don't understand the precedence rules of the destructuring sugar to get it to actually work

Reply:
Ah it looks like `settings.column()`'s result includes a `_numberFormatter` property which is taking precedence over the other settings. This will work, but I think leaving as-is is fine too.

```js
...settings.column(cols[metricIndex]),
_numberFormatter: undefined,
```

Reply:
I feel like my WTF factor would be less with the explicit single thing, as opposed to using the sugar and then explicitly unsetting a thing